charts/nmrxiv-app/templates/secret.yml

```diff
@@ -15,6 +14,15 @@ data:
   {{- else if .Values.mysql.enabled }}
   DB_PASSWORD: {{ required "A valid .Values.postgresql.postgresqlPassword entry required!" .Values.postgresql.postgresqlPassword |b64enc | quote }}
   {{- end -}}
+  {{- if .Values.redis.enabled }}
+  REDIS_PASSWORD: {{ required "A valid .Values.redis.auth.password entry required!" .Values.redis.auth.password |b64enc | quote }}
+  {{- end -}}
+  {{- if .Values.rabbitmq.enabled }}
+  RABBITMQ_PASSWORD: {{ required "A valid .Values.rabbitmq.auth.password entry required!" .Values.rabbitmq.auth.password |b64enc | quote }}
+  {{- end -}}
+  {{- if .Values.meilisearch.enabled }}
+  MEILISEARCH_KEY: {{ required "A valid .Values.meilisearch.secretKey entry required!" .Values.meilisearch.secretKey |b64enc | quote }}
+  {{- end -}}
   {{- if .Values.oauthProperties.enabled }}
   TWITTER_CLIENT_ID: {{ required "A valid .Values.oauthProperties.twitterClientID entry required!" .Values.oauthProperties.twitterClientID |b64enc | quote }}
   TWITTER_CLIENT_SECRET: {{ required "A valid .Values.oauthProperties.twitterClientID entry required!" .Values.oauthProperties.twitterClientSecret |b64enc | quote }}
```

Comment:
I would split the secrets into parts (two parts for now, services and Auth credentials [Github, Twitter]). I had a problem with the size of the information in the secret in my case. There is a limit on the number of keys for a secret. But if you don't get any errors yet, then everything is fine, for now.

Reply:
Thanks for the review..!! Yes as of now I did not get any error for secrets. But, yeah I will check this option for the future.